java/125-Valid-Palindrome.java

```diff
@@ -6,4 +7,30 @@ public boolean isPalindrome(String s) {
         content = new StringBuilder(content.toString().toLowerCase());
         String value = content.toString();
         return value.equals(content.reverse().toString());
+}
+
+
+//         Method 2
+//         Using Array
+//         38% more fast than the above code
+class Solution {
+    public boolean isPalindrome(String s) {
+        boolean flag = true;
```

Comment:
declaring this variable up here is unnecessary, we can `return false` on line 30 (there's no reason to keep iterating when we encounter a mismatched character), and `return true` on line 34.